keanu-project/src/main/java/io/improbable/keanu/util/Py4jByteArrayConverter.java

```diff
@@ -40,4 +40,20 @@
         byte[] byteArray = booleanBuffer.array();
         return byteArray;
     }
+
+    public static double[] toDoubleArray(byte[] byteArray, int itemSizeBytes){
```

Comment:
So I was actually using this before and the tests were failing because the size of integers in python was different to what the JVM considers an integer. I therefore used this approach for integers and additionally used it for doubles for consistency and in case the same problem occurred with python floats and differing sizes. 
@migwellian does that make sense?

Reply:
I see - looks like Python (numpy?) ints are 8 bytes.

So how about we have separate methods `toIntegerArray` and `toLongArray` with hard-coded item sizes, and from Python you call `toLongArray`?